modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/configuration/DistributionZonesHighAvailabilityConfiguration.java

```diff
@@ -61,32 +68,33 @@ public void start() {
     void startAndInit() {
         start();
 
-        updateSystemProperties(systemDistributedConfig.value());
+        updateSystemProperties(systemDistributedConfig.value(), 1);
     }
 
     /** Returns partition group reset timeout after a partition group majority loss. */
```

Comment:
The javadoc sounds a bit odd to me.

Reply:
I used the " timeout after a partition group majority loss." in all docs for this property. Do you want to rephrase it everywhere?

Reply:
I believe that Mirza means that it sounds like  "return ... after ... group majority loss". Are we going to return the value only after the fact that majority loss? ;)

Reply:
I see, guys. Fixed )